arctic/store/version_store.py

```diff
@@ -360,6 +360,27 @@ def _get_info(self, symbol, as_of=None):
         else:
             return """Handler: %s\n\nVersion document:\n%s""" % (handler.__class__.__name__, pprint.pformat(_version))
 
+    def get_data_info(self, symbol):
```

Comment:
`get_info`, should take `as_of`, as the person using this will have a `VerisonedItem` in their hand which will have a : `(symbol, version_number)` in it.